server/http.go

```diff
@@ -238,8 +238,19 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, webh
 		return
 	}
 
+	start := time.Unix(0, post.CreateAt*int64(time.Millisecond))
+	length := int((post.UpdateAt - post.CreateAt) / 60)
+	startText := start.Format("Mon Jan 2 15:04:05 -0700 MST 2006")
+
+	slackAttachment := model.SlackAttachment{
+		Fallback: fmt.Sprintf("Meeting has ended: started at %s, lenght: %d minute(s).", startText, length),
+		Title:    "Meeting Summary",
+		Text:     fmt.Sprintf("Date: %s\n\nMeeting Length: %d minute(s)", startText, length),
+	}
+
 	post.Message = "Meeting has ended."
```

Comment:
Do you mind updating this text so it is the same as the custom type? (i.e. `I have ended the meeting`)

Reply:
Done